Sources/Lang.php

```diff
@@ -17,6 +17,7 @@
 
 use SMF\Cache\CacheApi;
 use SMF\Localization\MessageFormatter;
+use ValueError;
```

Comment:
If we are adding this, then we might as well replace all instances of `\ValueError` with just `ValueError` in the rest of the file.

Reply:
I could go either way.  Just an IDE warning while I was looking at it. 

What is the preferred method here for PHP's built-ins?  Use \ or define the use?

Reply:
I prefer using fully qualified names (e.g. `\ValueError`) for built-in classes, for two reasons. 

1. There is less to maintain. Creating aliases of classes in the global namespace in the local namespace via `use` statements accomplishes very little except requiring us to spend more time keeping track of `use` statements.
2. It makes the code clearer. When I see a FQN in the code, I can immediately know that I am dealing with a built-in class, without having to waste any time scrolling to the top of the file to see if there are any unexpected class aliases in play.

Reply:
Thats fine with me.  Pick a standard.  So for built-ins,  we use \, for all others we define the use.